README.md

```diff
@@ -0,0 +1,57 @@
+# Backend
+
+## Using the backend for the first time
+
+After cloning the backend to your local repository you need to perform the following steps, in order to start start the flask application.
+
+<ol>
+    <li>
+        Create a python virtual environment (venv) in your backend directory:
+        <ul>
+            <li>
+                (terminal): python3 -m venv .venv or python -m venv .venv
+            </li>
+        </ul>
+    </li>
+    <li>
+        Install all necessary dependencies with the provided requirements.txt file:
+        <ol>
+            <li>
+                Change into the newly created virtual environment:
+                <ul>
+                    <li>
+                        (terminal): cd .venv
+                    </li>
+                </ul>
+            </li>
+            <li>
+                Activate the enviroment
+                <ul>
+                    <li>
+                        (terminal): source bin/activate
```

Comment:
This command only works for mac/linux. 
Windows users need to execute `.\Scripts\Activate.ps1` after changing into the .venv directory.

Reply:
Nice hint, i added your suggestion to the readme